User Guide/Projects.md

```diff
@@ -746,6 +769,14 @@ How to **publish** via **HTTP Post**:
 **Step 4:** Set msg as **Body**
 
 ![Image not Available](/assets/Fig104.png)
+
+Here are the steps to publish a topic using HTTP REST in ADEX application:
+
+- Identify the endpoint: You need to know the endpoint URL of the server that is configured to receive the message. This URL will typically include the protocol (e.g., http:// or https://), the hostname or IP address of the server, and a path to the resource that will receive the message.	
+- Define the message payload: You need to define the message payload that will be sent in the HTTP POST request. The payload will typically be a JSON or XML object that contains the message data as well as any additional metadata that is required.
+- Set the HTTP headers: You need to set the HTTP headers for the request, including the Content-Type header that indicates the format of the message payload (e.g., application/json).
+- Send the HTTP POST request to send the HTTP POST request to the server.
+- If ADEX topic message payload is a JSON object that contains the topic name ("my/topic") and the message content ("Hello, world!"). The HTTP POST request is sent to the "ADEX Portal" endpoint with the Content-Type header set to "application/json". Once the request is sent, the ADEX application logs a success message if the request is successful, or an error message if there is an error.
```

Comment:
'message content ("Hello, world!")' -> 'message content {"Hello": "World!"}'
"the ADEX application logs a success" -> "the system logs a success"

Reply:
done

Reply:
Removed "ADEX Portal"